### ISSUE TITLE
[dev/api] Some Serialization tests are failing for System.Collections.Specialized netcoreapp1.1 runs

### ISSUE THREAD

**Issue by joperezr** [dotnet/corefx#11313]
cc: @stephentoub 

```
call CoreRun.exe xunit.console.netcore.exe System.Collections.Specialized.Tests.dll  -xml testResults.xml -notrait Benchmark=true  -notrait category=OuterLoop -notrait category=failing -notrait category=nonwindowstests
xUnit.net console test runner (64-bit .NET Core)
Copyright (C) 2014 Outercurve Foundation.

Discovering: System.Collections.Specialized.Tests
Discovered:  System.Collections.Specialized.Tests
Starting:    System.Collections.Specialized.Tests
   System.Collections.Specialized.Tests.ListDictionaryKeysTests.IGenericSharedAPI_SerializeDeserialize(count: 0) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Specialized.ListDictionary+NodeKeyValueCollection' in Assembly 'System.Collections.Specialized, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(19,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.ListDictionaryKeysTests.IGenericSharedAPI_SerializeDeserialize(count: 1) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Specialized.ListDictionary+NodeKeyValueCollection' in Assembly 'System.Collections.Specialized, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.HybridDictionary_CaseSensitive_Tests.IGenericSharedAPI_SerializeDeserialize(count: 0) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Hashtable' in Assembly 'System.Collections.NonGeneric, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Type objectType, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Write(WriteObjectInfo objectInfo, NameInfo memberNameInfo, NameInfo typeNameInfo)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.ListDictionaryKeysTests.IGenericSharedAPI_SerializeDeserialize(count: 75) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Specialized.ListDictionary+NodeKeyValueCollection' in Assembly 'System.Collections.Specialized, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.HybridDictionaryKeysTests.IGenericSharedAPI_SerializeDeserialize(count: 0) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Specialized.ListDictionary+NodeKeyValueCollection' in Assembly 'System.Collections.Specialized, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.HybridDictionaryKeysTests.IGenericSharedAPI_SerializeDeserialize(count: 1) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Specialized.ListDictionary+NodeKeyValueCollection' in Assembly 'System.Collections.Specialized, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.HybridDictionary_CaseSensitive_Tests.IGenericSharedAPI_SerializeDeserialize(count: 1) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Hashtable' in Assembly 'System.Collections.NonGeneric, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Type objectType, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Write(WriteObjectInfo objectInfo, NameInfo memberNameInfo, NameInfo typeNameInfo)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.HybridDictionaryKeysTests.IGenericSharedAPI_SerializeDeserialize(count: 75) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Hashtable+KeyCollection' in Assembly 'System.Collections.NonGeneric, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.HybridDictionary_CaseSensitive_Tests.IGenericSharedAPI_SerializeDeserialize(count: 75) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Hashtable' in Assembly 'System.Collections.NonGeneric, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Write(WriteObjectInfo objectInfo, NameInfo memberNameInfo, NameInfo typeNameInfo)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.HybridDictionary_CaseInsensitive_Tests.IGenericSharedAPI_SerializeDeserialize(count: 0) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Hashtable' in Assembly 'System.Collections.NonGeneric, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Type objectType, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Write(WriteObjectInfo objectInfo, NameInfo memberNameInfo, NameInfo typeNameInfo)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.HybridDictionary_CaseInsensitive_Tests.IGenericSharedAPI_SerializeDeserialize(count: 1) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Hashtable' in Assembly 'System.Collections.NonGeneric, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Type objectType, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Write(WriteObjectInfo objectInfo, NameInfo memberNameInfo, NameInfo typeNameInfo)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.HybridDictionary_CaseInsensitive_Tests.IGenericSharedAPI_SerializeDeserialize(count: 75) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Hashtable' in Assembly 'System.Collections.NonGeneric, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Write(WriteObjectInfo objectInfo, NameInfo memberNameInfo, NameInfo typeNameInfo)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.HybridDictionaryValuesTests.IGenericSharedAPI_SerializeDeserialize(count: 0) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Specialized.ListDictionary+NodeKeyValueCollection' in Assembly 'System.Collections.Specialized, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.HybridDictionaryValuesTests.IGenericSharedAPI_SerializeDeserialize(count: 1) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Specialized.ListDictionary+NodeKeyValueCollection' in Assembly 'System.Collections.Specialized, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.HybridDictionaryValuesTests.IGenericSharedAPI_SerializeDeserialize(count: 75) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Hashtable+ValueCollection' in Assembly 'System.Collections.NonGeneric, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.ListDictionaryValuesTests.IGenericSharedAPI_SerializeDeserialize(count: 0) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Specialized.ListDictionary+NodeKeyValueCollection' in Assembly 'System.Collections.Specialized, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.ListDictionaryValuesTests.IGenericSharedAPI_SerializeDeserialize(count: 1) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Specialized.ListDictionary+NodeKeyValueCollection' in Assembly 'System.Collections.Specialized, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
   System.Collections.Specialized.Tests.ListDictionaryValuesTests.IGenericSharedAPI_SerializeDeserialize(count: 75) [FAIL]
      System.Runtime.Serialization.SerializationException : Type 'System.Collections.Specialized.ListDictionary+NodeKeyValueCollection' in Assembly 'System.Collections.Specialized, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' is not marked as serializable.
      Stack Trace:
            at System.Runtime.Serialization.FormatterServices.GetSerializableFields(Type type)
            at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitMemberInfo()
            at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
            at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, Header[] inHeaders, BinaryFormatterWriter serWriter, Boolean fCheck)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph, Header[] headers, Boolean check)
            at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
         F:\git\corefx\src\Common\tests\System\Collections\IEnumerable.NonGeneric.Serialization.Tests.cs(25,0): at System.Collections.Tests.IEnumerable_NonGeneric_Tests.IGenericSharedAPI_SerializeDeserialize(Int32 count)
Finished:    System.Collections.Specialized.Tests

=== TEST EXECUTION SUMMARY ===
   System.Collections.Specialized.Tests  Total: 2106, Errors: 0, Failed: 18, Skipped: 0, Time: 0.453s
Finished running tests.  End time=15:57:56.05, Exit code = 18
```

**Comment by stephentoub** [comment]
Temporarily blocked until we update the corefx packages on which we depend....

**Comment by stephentoub** [comment]
Fixed by #11396
